pom.xml

```diff
@@ -4369,6 +4369,7 @@
                 <avoidCallsTo>com.puppycrawl.tools.checkstyle.utils.UnmodifiableCollectionUtil</avoidCallsTo>
               </avoidCallsTo>
               <excludedMethods>
+                <param>format</param>
```

Comment:
I meant calculateBufferLength

Reply:
sorry 

Done

Reply:
Not done. it is not mutated because of method calculateBufferLength. 
If I put the method calculateBufferLength then only this method will be ignored. format method is implemented only in 2 classes 
1) in this AuditEventDefaultFormatter 
2) https://github.com/checkstyle/checkstyle/blob/9e931263c2eb441fa7f893001e18005b1a2097a5/src/test/java/com/puppycrawl/tools/checkstyle/internal/utils/AuditEventUtFormatter.java#L35 --> for which we are not doing any mutation testing.

I have also tried to use `<avoidCallsTo>java.lang.StringBuilder</avoidCallsTo>`  This is also not working.